app/models/command.rb

```diff
@@ -41,6 +41,15 @@ class Command < Hashie::Dash
   property :confirmation, default: nil
   property :scripts,      default: []
   property :has_context,  default: true
+
+  # Flags the output to be displayed sequentially, the jobs will still run
+  # concurrently
+  property :sequential_output, default: false
```

Comment:
I can't see any reason for us to have this additional complexity here.  If people want the output to not be interleaved they can either pipe the output through `sort` or event better through `dshbak`.

Am I missing something here?

Reply:
Removed in rebase